### DISCUSSION TITLE
Win10x64 no inet after interface turned off and then on

### DISCUSSION THREAD

**Discussion by necros2k7** [DNSCrypt/dnscrypt-proxy#1703] [Potential issues]
after these commands no inet reconnection happens:
netsh interface set interface "Ethernet" DISABLED
netsh interface set interface "Ethernet" ENABLED
it happens after couple of minutes,
with dns-crypt off everything is OK, how to troubleshoot?
Machine uses Simple dns-crypt 0.7.1 and 2.0.45 binary

**Comment by lifenjoiner** [comment]
> no inet reconnection

Explain what it is exactly ... No internet completely, [NCSI](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Windows-NCSI) related, or anything else?

**Comment by necros2k7** [comment]
Browsers can`t open pages, Messengers don`t reconnect, I relate it to slow dnscrypt reconnection, in couple of minutes after turning on network interface everything is ok
